### PR TITLE
make assertions on packaging, recommend tox for all testings

### PR DESCRIPTION
**Pull request by gaborbernat**
Resolves #909.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -91,16 +91,16 @@ commands = pip wheel -w {envtmpdir}/build --no-deps .
...
-changedir = {toxinidir}/bin
+changedir = {toxinidir}/tasks
 extras =
-commands = python rebuild-script.py
+commands = python update_embedded.py
 
 [testenv:upgrade]
 description = upgrade pip/wheels/setuptools to latest
 skip_install = true
 extras = pip >= 18.1.0
```
Comment (webknjaz) [inline tox.ini]: @gaborbernat this is what I was talking about on Twitter

Comment (gaborbernat) [reply tox.ini]: that's indeed a bug 😅

Comment (webknjaz) [reply tox.ini]: PR incoming

Comment (gaborbernat) [reply tox.ini]: should be deps instead

Comment (webknjaz) [reply tox.ini]: Sure :)

**Pull request merged**